docker/DOCKERFILE

```diff
@@ -0,0 +1,11 @@
+LABEL maintainer=omelisra123@gmail.com
```

Comment:
Using labels to better understand the docker image is always a good idea.
However, it would be even better if you include more labels like build date etc.
https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d